kfs-tax/src/main/java/edu/arizona/kfs/module/tax/batch/BatchPayeeFormStep.java

```diff
@@ -0,0 +1,102 @@
+package edu.arizona.kfs.module.tax.batch;
+
+import java.io.BufferedOutputStream;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.util.Date;
+import java.util.List;
+
+import org.kuali.kfs.sys.batch.AbstractStep;
+
+import edu.arizona.kfs.module.tax.TaxConstants;
+import edu.arizona.kfs.module.tax.businessobject.Payee;
+import edu.arizona.kfs.module.tax.businessobject.Payer;
+import edu.arizona.kfs.module.tax.service.TaxForm1099GeneratorService;
+import edu.arizona.kfs.module.tax.service.TaxParameterHelperService;
+import edu.arizona.kfs.module.tax.service.TaxPayeeService;
+import edu.arizona.kfs.sys.KFSConstants;
+
+public class BatchPayeeFormStep extends AbstractStep {
+    private static final org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(BatchPayeeFormStep.class);
+    private TaxPayeeService taxPayeeService;
+    private TaxParameterHelperService taxParameterHelperService;
+    private TaxForm1099GeneratorService taxForm1099GeneratorService;
+
+    public void setTaxPayeeService(TaxPayeeService taxPayeeService) {
+        this.taxPayeeService = taxPayeeService;
+    }
+
+    public void setTaxParameterHelperService(TaxParameterHelperService taxParameterHelperService) {
+        this.taxParameterHelperService = taxParameterHelperService;
+    }
+
+    public void setTaxForm1099GeneratorService(TaxForm1099GeneratorService taxForm1099GeneratorService) {
+        this.taxForm1099GeneratorService = taxForm1099GeneratorService;
+    }
+
+    @Override
+    public boolean execute(String jobName, Date jobRunDate) throws InterruptedException {
+        int year = taxParameterHelperService.getTaxYear();
+        List<Payee> list = taxPayeeService.loadPayees(Integer.valueOf(year));
+        boolean success = generateBatchPayeeForms(year, list);
+        return success;
+
+    }
+
+    private boolean generateBatchPayeeForms(int year, List<Payee> payeeList) {
```

Comment:
You obviously misunderstood my request. The method containing business logic that I wanted moved to a service is generateBatchPayeeForms. The execute method was appropriate as originally coded. The only change I would have expected in the execute method was how the generateBatchPayeeForms method was called:
`boolean success = taxForm1099GeneratorService.generateBatchPayeeForms(year, list);`